family2family/src/main/java/org/cbtf/f2f/model/ChildDiagnosisRepository.java

```diff
@@ -1,10 +0,0 @@
-package org.cbtf.f2f.model;
```

Comment:
I realized that while these classes are not needed for showing the table in the console, they are needed for displaying the tables on http://localhost:8080/internal-api .